src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -1740,10 +1741,10 @@ namespace HeatBalanceSurfaceManager {
 			e.DividerTempSurfIn = 23.0;
 			e.DividerTempSurfInOld = 23.0;
 			e.DividerTempSurfOut = 23.0;
-
+		
```

Comment:
Stray whitespace

Reply:
I can look at both of them. Working on it right now.